cpp/include/cuml/fil/fil.h

```diff
@@ -240,6 +243,9 @@ struct treelite_params_t {
   float threshold;
   // storage_type indicates whether the forest should be imported as dense or sparse
   storage_type_t storage_type;
+  // blocks_per_sm, if set, works as a limit to improve cache hit rate for larger forests
+  // suggested values (if nonzero) are from 2 to 7
+  int blocks_per_sm = 0;
```

Comment:
It is important to keep the existing Python code working out of the box. Regarding the C++ code, if a new field is added to a structure, then the user is responsible for initializing it. In case a C++ function is used in Python, the Python wrapper should initialize the field appropriately. For the C++ tests, appropriate initialization should be added.